### PR TITLE
feat(RHIN-830):Add ZeroState to policys

### PR DESCRIPTION
**Pull request by adonispuente**
As in accordance to these mocks, this PR adds zero state to Policies , 

https://docs.google.com/document/d/1FGuzbL2jyRmn_coYD5Z_o3d0MhOf1vQDT46NQvsfPyA/edit#heading=h.1hvxoje16w3i  

https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/a/qbe9Ol2#Inspect

### CODE REVIEW

**Comment by josejulio** [comment]
According to the docs:
```
Show When:
Account has 0 systems connected with Insights-Client/RHC OR RHSM (ex: nothing in inventory)
```

What is currently doing is showing the `zero-state` when there are no policies - is that expected?
I'm guessing an additional check for the systems needs to be done?

**Comment by adonispuente** [comment]
> 

Youre correct, I was told they were the same but the request definitely needs to be to /hosts. There is an optimization coming down the way, but the current solution is the make the api call inside the app. (eventually it will be from chrome and only called once)

**Comment by adonispuente** [comment]
@josejulio The PM for Policies should be aware of this and have approved it as thats my understanding for all the apps that have mocks.

It does seem odd to just remove it all together. I'll reach out to UX/PM is in charge of the mocks in regards to that and the other edge-case you mentioned.

File: src/pages/ListPage/ListPage.tsx [hunk 1]
```diff
@@ -55,12 +57,28 @@ const getPoliciesFromPayload = (payload: ReturnType<typeof useGetListPagePolicie
...
+        try {
+            axios
+            .get(`${INVENTORY_TOTAL_FETCH_URL}?page=1&per_page=1`)
+            .then(({ data }) => {
+                setHasSystems(data.total > 0);
+            });
+        } catch (e) {
+            console.log(e);
+        }
+    }, [ hasSystems ]);
```
Comment (josejulio) [inline src/pages/ListPage/ListPage.tsx:80]: You don't need this dependency.

```suggestion
    }, [ setHasSystems ]);
```

File: src/pages/ListPage/ListPage.tsx [hunk 2]
```diff
@@ -194,48 +212,59 @@ const ListPage: React.FunctionComponent<unknown> = () => {
...
-                                error={ policyTableErrorValue }
-                                onSort={ sort.onSort }
-                                sortBy={ sort.sortBy }
-                                linkToDetailPolicy={ true }
-                            />
-                        </PolicyToolbar>
-                    </Section>
-                )}
+
+                ) : getPoliciesQuery.hasPolicies === false && hasSystems  ?
```
Comment (josejulio) [inline src/pages/ListPage/ListPage.tsx:224]: I think this `hasSystems` is redundant - it wasn't `false` in previous condition.

```suggestion
                ) : getPoliciesQuery.hasPolicies === false ?
```

Comment (adonispuente) [reply src/pages/ListPage/ListPage.tsx:224]: Fair enough

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**